src/dashboard.ts

```diff
@@ -85,6 +85,16 @@ export class Dashboard extends embed.Embed implements IDashboardNode {
      * Validate load configuration.
      */
     validate(config: models.IDashboardLoadConfiguration): models.IError[] {
-      return models.validateDashboardLoad(config);
+      let error = models.validateDashboardLoad(config);
+      return error ? error : this.ValidatePageView(config.pageView);
```

Comment:
I think it will return false or true like this - won't it? I want to return the error itself.